### ISSUE TITLE
Same server links to non-wiki pages are converted into useless links

### ISSUE THREAD

**Issue by StarArmy** [cosmocode/dokuwiki-plugin-interwikipaste#2]
If you have a wiki using this plugin at https://example.com/wiki and a forum at https://example.com/forum then the plugin will convert the https://example.com/forum/threads/1234 to :forum:threads:1234 which is a broken link.

**Comment by bvdbos** [comment]
I would think one can will only paste valid URL's to existing pages on the local wiki. So maybe do a check before rewriting the url?

**Comment by arlt** [comment]
@StarArmy do you have a configuration file `conf/interwiki.local.conf` with e.g.?!:
```
wiki https://example.com/wiki/{NAME}
```

**Comment by StarArmy** [comment]
> 
> 
> @StarArmy do you have a configuration file `conf/interwiki.local.conf` with e.g.?!:
> 
> ```
> wiki https://example.com/wiki/{NAME}
> ```

Yes, I am using what appears to be the default file that comes with vanilla Dokuwiki.

**Comment by arlt** [comment]
have you tried to extend it with:

```
forum https://example.com/forum/{NAME}
```

**Comment by arlt** [comment]
> Yes, I am using what appears to be the default file that comes with vanilla Dokuwiki.

Are you sure there is an entry for your own wiki?
I assume that one must configure both entries. Your wiki and your forum.

**Comment by StarArmy** [comment]
I don't think you understand the issue, if a user pastes a URL from anywhere on my site that's not the wiki, the link is turned into a broken link by this plugin. The problem is that the plugin isn't able to tell what is in the wiki or not in the wiki when the domain is the same.

**Comment by arlt** [comment]
I did understand your problem.
